api/src/test/java/gov/cms/ab2d/api/controller/MaintenanceModeAPITests.java

```diff
@@ -38,13 +39,18 @@
     @Autowired
     private PropertiesService propertiesService;
 
+    @Autowired
+    DoAll doAll;
+
     @AfterEach
     void tearDown() {
         PropertiesDTO propertiesDTO = new PropertiesDTO();
         propertiesDTO.setKey(MAINTENANCE_MODE);
         propertiesDTO.setValue("false");
 
         propertiesService.updateProperties(Collections.singletonList(propertiesDTO));
```

Comment:
Does this not log to an event table at any point?

Reply:
Virtually all the mockmvc calls result in events being logged.  They can also commonly result in jobs being created.